components/sensor/qmc5883l.rst

```diff
@@ -0,0 +1,75 @@
+QMC5883L Magnetometer
+=====================
+
+.. seo::
+    :description: Instructions for setting up QMC5883L IMU compass sensors.
+    :image: qmc5883l.jpg
+    :keywords: QMC5883L
+
+The ``qmc5883l`` allows you to use your QMC5883L triple-axis magnetometers
+(`datasheet <http://wiki.sunfounder.cc/images/7/72/QMC5883L-Datasheet-1.0.pdf>`__) with
+ESPHome. The :ref:`I²C Bus <i2c>` is
+required to be set up in your configuration for this sensor to work.
```

Comment:
Please add a short paragraph below this one explaining the difference between this sensor and the HMC5883L.